src/questionnaireitem_media_video.js

```diff
@@ -59,7 +59,23 @@ QuestionnaireItemMediaVideo.prototype._createMediaNode = function() {
 
     this.videoNode = document.createElement('video');
     this.videoNode.oncanplaythrough = this._onloaded.bind(this);
-    this.videoNode.src = this.url;
+
+    if (Array.isArray(this.url)) {
```

Comment:
This changes the implementation approach as `this.url` is inherited from QuestionnaireItemMedia and it defined as one url only.
Thus QuestionnaireItemMediaVideo would use this in a different, manner which might be confusing.

I would recommend:
* changing the QuestionnaireItemMedia (and all it's childs), so that multiple URLs are supported (ie. make this.url an Array of string).
* for *Image only allow one URL by providing a warning if more than one are provided and only take the first one.
* In the constructor of QuestionnaireItemMedia: allow url to be string as well as Array of string (update documentation and use `this.url = Array.isArray(url) ? url : [url]`.
In this way it must not be distinguished later.
* update createUI() code for all childs of QuestionnaireItemMedia